scripts/detect_artefacts.py

```diff
@@ -0,0 +1,105 @@
+# Copyright (C) 2021, Mindee.
+
+# This program is licensed under the Apache License version 2.
+# See LICENSE or go to <https://www.apache.org/licenses/LICENSE-2.0.txt> for full license details.
+
+
+import os
+
+os.environ['USE_TORCH'] = '1'
+
+import argparse
+import logging
+
+import cv2
+import matplotlib.pyplot as plt
+from matplotlib.pyplot import figure
+import torch
+
+from doctr.models import obj_detection
+
+
+def inference(artefact, root_dir, test_list, visualize):
```

Comment:
the function is not doing inference --> "plot_predictions'

Reply:
Thanks, alright .